feature/community/src/main/java/com/iteneum/community/presentation/CommunityComposable.kt

```diff
@@ -0,0 +1,72 @@
+package com.iteneum.community.presentation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.*
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.iteneum.community.R
+import com.iteneum.community.domain.CardMenuItem
+import com.iteneum.community.domain.CardModel
+import com.iteneum.designsystem.components.LpIconTextCard
+import com.iteneum.designsystem.theme.LeasePertTheme
+
+/**
+ * This function creates a view that displays a list of community sections.
+ * @param viewModel receives a viewModel to manage the logic of the view
+ *this view model is used to handle the loading of the components of the interface
+ * including icons, text and button click functionality.
+ *
+ * @author Andrés Ivan Medina Herrera
+ */
+@Composable
+fun CommunitylistUI() {
+    val cardWith = LeasePertTheme.sizes.extraSize124
+    val cardHeight = LeasePertTheme.sizes.extraSize128
+    val cardPadding8 = LeasePertTheme.sizes.smallerSize
+    val cardPadding15 = LeasePertTheme.sizes.smallSize
+    val cardPadding10 = LeasePertTheme.sizes.midSmallSize
+    val cardPadding20 = LeasePertTheme.sizes.minorRegularSize
+    val communityCardButtons = listOf(
+        CardModel(Icons.Outlined.Diamond, CardMenuItem.AMENITIES),
+        CardModel(Icons.Outlined.Event, CardMenuItem.EVENTS),
+        CardModel(Icons.Outlined.QuestionAnswer, CardMenuItem.COMMUNITY_WALL),
+        CardModel(Icons.Outlined.LibraryBooks, CardMenuItem.DOITYOURSELF),
+        CardModel(Icons.Outlined.Store, CardMenuItem.SERVICES)
+    )
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(top = cardPadding20, start = cardPadding10, end = cardPadding10)
+    ) {
+        Text(
+            text = "Community",
```

Comment:
I think this text we can add it to the string resource

Reply:
What is the standard used in the other screens?   as this is a Type of thext that exists in other screens there should be a standard is not it?  @irv205